test/boxtype.cxx

```diff
@@ -176,6 +177,7 @@ int main(int argc, char ** argv) {
   bt("FL_GTK_ROUND_DOWN_BOX",FL_GTK_ROUND_DOWN_BOX);
   bg->end();
   window->resizable(window);
+  Fl_Scheme_Choice scheme_choice(610, 10, 150, 30, "Scheme:");
```

Comment:
I don't feel that this is needed here, because the window shows most box types anyway, regardless of the selected scheme. Then again, why not.

Reply:
It was really needed. It's important to be able to switch the scheme and see which of the standard boxtypes (those of the "normal" scheme are overwritten - and how.

If you actually try it you may notice that there are no "round(ed)" boxes in the `gleam` scheme. This has the effect that radio buttons are not round (circles) as they should be. Try the "buttons" demo as well.